centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json

```diff
@@ -0,0 +1,1618 @@
+						"disableCookies": false,
+						"disableUrlEncoding": true
+					},
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "{{baseUrl}}/configuration/dashboards/{{dashboardID}}",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"configuration",
+								"dashboards",
+								"{{dashboardID}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		}
+	],
+	"auth": {
+		"type": "apikey",
+		"apikey": [
+			{
+				"key": "value",
+				"value": "{{token}}",
+				"type": "string"
+			},
+			{
+				"key": "key",
+				"value": "X-AUTH-TOKEN",
+				"type": "string"
+			}
+		]
+	},
+	"event": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					"",
+					"const moment = require('moment');",
+					"if (!pm.globals.get(\"loginScriptExecuted\") || isTokenExpired()) {",
+					"    pm.sendRequest({",
+					"        url: pm.variables.replaceIn('{{baseUriCentreonApi}}/{{version}}/login'),",
+					"        followRedirects: true,",
+					"        encodeUrl: true,",
+					"        useCookieJar: false,",
+					"        method: 'POST',",
+					"        header: {",
+					"            'Content-Type': 'application/json'",
+					"        },",
+					"        body: {",
+					"            mode: 'raw',",
+					"            raw: JSON.stringify({",
+					"                \"security\": {",
+					"                    \"credentials\": {",
+					"                        \"login\": pm.variables.get(\"api_login\"),",
+					"                        \"password\": pm.variables.get(\"api_password\")",
+					"                    }",
+					"                }",
+					"            })",
+					"        }",
+					"    }, function (err, res) {",
+					"        if (!err) {",
+					"            pm.globals.set(\"token\", res.json().security.token);",
+					"            pm.globals.set(\"tokenIssuedAt\", moment().toISOString());",
+					"            pm.globals.set(\"loginScriptExecuted\", true);",
+					"        } else {",
+					"            console.error(\"Error during login request:\", err);",
+					"            pm.globals.set(\"loginScriptExecuted\", false);",
+					"        }",
+					"    });",
+					"}",
+					"",
+					"function isTokenExpired() {",
+					"    const tokenIssuedAt = pm.globals.get(\"tokenIssuedAt\");",
+					"    if (tokenIssuedAt) {",
+					"        const issuedAtMoment = moment(tokenIssuedAt);",
+					"        const currentMoment = moment();",
+					"        const elapsedMinutes = currentMoment.diff(issuedAtMoment, 'minutes');",
+					"        return elapsedMinutes >= 60; // If one hour or more has elapsed, the token is considered expired.",
+					"    }",
+					"    return true; //If the variable is not defined, the token is considered expired.",
+					"}",
+					""
+				]
+			}
+		},
+		{
+			"listen": "test",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		}
+	],
+	"variable": [
```

Comment:
You can remove variable names in the collection to reduce the number of unused rows